style.css

```diff
@@ -106,4 +106,28 @@ a {
 
 .lng-badge-TypeScript {
     background-color: var(--typescript);
+}
+
+/* skeleton screen */
+
+.skeleton-card {
```

Comment:
As per my above commnent, this should contain only the stuff specific to the skeleton card. (no border-radius, no padding, etc).